Packs/Zimperium/ReleaseNotes/1_0_5.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Zimperium
+- Updated the **file** command, to return a different result for each indicator. 
```

Comment:
@DeanArbel @bakatzir Shouldn't this be with **breaking changes** like the other related PRs?

Reply:
If he is updating the `Base` pack, it would not break BC.